alf/networks/networks.py

```diff
@@ -308,3 +315,184 @@ def forward(self, input, state):
                 lambda x: x.float() if x.dtype.is_floating_point else x, input)
         with torch.cuda.amp.autocast(self._enabled):
             return self._net(input, state)
+
+
+@alf.configurable
+@alf.repr_wrapper
+class NoisyFC(Network):
```

Comment:
The name hints this class is a layer but in fact it's a network. Maybe changed to a better name? The original paper uses network.

Reply:
The network in the original paper is really a network consisting of several FC layers.